community/lucene-index/src/main/java/org/neo4j/kernel/api/impl/index/LuceneAllDocumentsReader.java

```diff
@@ -37,7 +42,14 @@ public LuceneAllDocumentsReader(
             LuceneIndexAccessor.LuceneReferenceManager<IndexSearcher> searcherManager )
     {
         this.searcherManager = searcherManager;
-        this.searcher = searcherManager.acquire();
+        try
+        {
+            this.searcher = searcherManager.acquire();
+        }
+        catch ( IOException e )
+        {
+            throw new RuntimeException( e );
```

Comment:
Is RuntimeException the appropriate exception to throw here? IOException or some other checked exception would make things more explicit.